### PR TITLE
sdk/java: add ability to update account/asset tags

### PR DESCRIPTION
**Pull request by jeffomatic**

### CODE REVIEW

**Comment by jeffomatic** [comment]
PTAL @boymanjor @dominic

File: sdk/java/src/main/java/com/chain/api/Account.java [hunk 1]
```diff
@@ -255,6 +266,66 @@ public Builder setTags(Map<String, Object> tags) {
   }
 
   /**
+   * Use this class to update an account's tags.
+   */
+  public static class TagUpdateBuilder {
+    @SerializedName("alias")
```
Comment (boymanjor) [inline sdk/java/src/main/java/com/chain/api/Account.java]: nit: Are the @SerializedName annotations necessary?

File: sdk/java/src/test/java/com/chain/integration/UpdateTagsTest.java [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            .addFilterParameter(account2.id)
+            .execute(client);
+
+    List<Account> accountList = new ArrayList<>();
+    while (accounts.hasNext()) {
+      accountList.add(accounts.next());
+    }
+
+    assertEquals(accountList.get(0).tags.get("y"), "three");
+    assertEquals(accountList.get(1).tags.get("x"), "three");
```
Comment (boymanjor) [inline sdk/java/src/test/java/com/chain/integration/UpdateTagsTest.java]: nit: can we add a comment about the query results being in reverse chron. order of creation? I first questioned if we could just rely on this ordering until I checked.

**Comment by boymanjor** [comment]
LGTM with some nits.

File: sdk/java/src/main/java/com/chain/api/Account.java (hunk 1 above)
Comment (jeffomatic) [inline sdk/java/src/main/java/com/chain/api/Account.java]: Nice catch

File: sdk/java/src/test/java/com/chain/integration/UpdateTagsTest.java (hunk 2 above)
Comment (jeffomatic) [inline sdk/java/src/test/java/com/chain/integration/UpdateTagsTest.java]: Good call

**Pull request merged**